content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md

```diff
@@ -110,29 +116,46 @@ Therefore, the first step is to remove some files and content from the files in
 
    All of the pallets used in a runtime must be set to compile with the `no_std` features.
 
-1. Add a skeleton set of pallet dependencies and [macros](/reference/frame-macros) that the custom pallet requires by copying the following code:
+1. Add the following skeleton code that includes a few minimal [macros for FRAME V2](https://paritytech.github.io/substrate/master/frame_support/attr.pallet.html):
 
    ```rust
+   //! Substrate Proof-of-Existence Pallet
+   #![cfg_attr(not(feature = "std"), no_std)]
+
```

Comment:
```suggestion
```
This is in step 1 already.

Reply:
I see the style Lisa is using is _super extra_ explicit in steps... think we should pull this into it's own step? IMHO no.